features/importing/work_import.feature

```diff
@@ -220,3 +220,11 @@ Feature: Import Works
     When I import "http://www.intimations.org/fanfic/idol/Huddling.html"
     Then I should see "Preview"
       And I should see "English"
+
+  Scenario: Import and check that redirects work
+    When I import "http://www.scarvesandcoffee.net/viewstory.php?sid=9570"
+      And I press "Post"
+      And I go to the redirect page
+      And I fill in "original_url" with "http://www.scarvesandcoffee.net/viewstory.php?sid=9570"
```

Comment:
Could you use the label text here instead, i.e. `And I fill in "Original URL of work" with "http://www.scarvesandcoffee.net/viewstory.php?sid=9570"`? It's nice to have feature tests correspond with what the user actually sees. (Also, a better scenario description might be "Searching for an imported work by URL will redirect you to the work," so if the test fails, we know to start by checking the search function rather than the redirect code.)